apps/web/components/Organization/Slice/TabSection/TabSection.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { TabSection } from '@island.is/web/graphql/schema'
 import { Box, GridColumn, GridRow, Tabs, Text } from '@island.is/island-ui/core'
-import * as styles from '@island.is/web/screens/AboutPage/AboutPage.treat'
+import * as styles from '@island.is/web/screens/Organization/Organization.treat'
```

Comment:
will this affect the style of the tab section?

Reply:
I created a file called Organization.treat since the TabSection was under Organization.
I copied and pasted the code from AboutPage.treat into Organization.treat, a new file.
I assumed this would not change the styles. Am I missing something?